### PR TITLE
409 lnurl UI

### PR DESCRIPTION
**Pull request by ubbabeck**
## About pr

This PR references #409 

 title, image location, button, no warning

- [x] Remove warning ce7b053e6a6b530dbfe6de1302141114904740ed
- [x] edit title
- [x] move metadata to bottom 56ade127c82b7db4655cb70705a9dd5764c30b62
- [x] change button to continue e724c829dcd958623b001bbebd71eaf79018ade0

Cosmetic fixes e5aabcd4fbf677ad00658ca152925908e159d401 and d5982f77f0f9f7e74a6ad473e577bec61c003619

### CODE REVIEW

**Comment by ubbabeck** [comment]
@kingonly would we like more or less space ?
## More space 
space
## less space
less_space

**Comment by kingonly** [comment]
@ubbabeck Same as breezmobile for now.

**Comment by ubbabeck** [comment]
> @ubbabeck Same as breezmobile for now.

Okay, the current state of the pr should be same as breezmobile.

File: lib/routes/lnurl/payment/lnurl_payment_page.dart [hunk 1]
```diff
@@ -170,14 +154,18 @@ class LNURLPaymentPageState extends State<LNURLPaymentPage> {
                     controller: _identifierController,
                   )
                 ],
+                LNURLMetadata({
```
Comment (erdemyerebasmaz) [inline lib/routes/lnurl/payment/lnurl_payment_page.dart:157]: Please add top padding to this widget to match Breezmobile UI.

File: lib/routes/lnurl/payment/lnurl_payment_page.dart [hunk 2]
```diff
@@ -129,10 +109,14 @@ class LNURLPaymentPageState extends State<LNURLPaymentPage> {
...
-                      (widget.requestData.minSendable ~/ 1000).toString(),
-                      (widget.requestData.maxSendable ~/ 1000).toString(),
+                  Padding(
+                    padding: const EdgeInsets.only(top: 8),
+                    child: Text(
+                      texts.lnurl_fetch_invoice_limit(
+                        (widget.requestData.minSendable ~/ 1000).toString(),
+                        (widget.requestData.maxSendable ~/ 1000).toString(),
+                      ),
+                      textAlign: TextAlign.left,
```
Comment (erdemyerebasmaz) [inline lib/routes/lnurl/payment/lnurl_payment_page.dart:119]: Please use the same text style with Breezmobile UI.
Breezmobile text theme values are likely to be deprecated and replaced with updated versions on c-Breez. Ping me in Slack to find the equivalent theme value.

File: lib/routes/lnurl/widgets/lnurl_metadata.dart [hunk 3]
```diff
@@ -13,14 +13,18 @@ class LNURLMetadata extends StatelessWidget {
...
         Text(
           metadataMap['text/long-desc'] ?? metadataMap['text/plain'],
-          style: Theme.of(context).textTheme.bodySmall,
+          style: Theme.of(context).textTheme.bodyLarge,
         ),
-        _LNURLMetadataImage(
-          base64String: base64String,
+        Container(
+          alignment: Alignment.center,
+          padding: const EdgeInsets.symmetric(vertical: 150),
```
Comment (erdemyerebasmaz) [inline lib/routes/lnurl/widgets/lnurl_metadata.dart:24]: Remove the padding here, or use a much acceptable value in the range of 8-24 and wrap the Container with an Expanded widget so it's centered in the available space.
There are lnurl payments where payee details need to be filled and this wouldn't look great on that case, you can see for yourself by reversing the conditions to display those widgets.

**Review by erdemyerebasmaz — CHANGES_REQUESTED** [review]
Good job!
There are some minor differences between breezmobile I've noticed and a layout feedback that should be addressed before merging.

File: lib/routes/lnurl/widgets/lnurl_metadata.dart (hunk 3 above)
Comment (ubbabeck) [reply lib/routes/lnurl/widgets/lnurl_metadata.dart]: Ran into some difficulties when trying wrap the container with an expanded widget. This is most likely due to the constrains being unbounded somewhere

**Review by erdemyerebasmaz — APPROVED** [review]

**Review by ubbabeck — COMMENTED** [review]
Thank you. Good solution seperating both metadata into two classes.

**Pull request merged**